Combobox.js

```diff
@@ -767,6 +875,22 @@ define([
 			};
 		}),
 
+		// HasDropDown#_dropDownKeyUpHandler() override.
+		// Do not call openDropDown if widget does not have a down arrow shown.
+		_dropDownKeyUpHandler: dcl.superCall(function (sup) {
+			return function () {
+				if (this.hasDownArrow) {
+					if (this.inputNode.value.length === this.minFilterChars === 0) {
+						this.list.query = this.getInitialQuery;
+					} else if (this.inputNode.value.length < this.minFilterChars) {
+						return;
+					}
+
+					sup.call(this);
+				}
+			};
+		}),
```

Comment:
Looks like Combobox-decl.html is missing a test for when hasDownArrow == false?   Then by corollary, you are missing an automated test for this piece of code?

I'm also confused by what this code is doing.  The comment says "Do not call openDropDown if widget does not have a down arrow shown.", but of course the dropdown is supposed to open at some point.   Can you make the method description clearer?